boards/st/stm32n6570_dk/doc/index.rst

```diff
@@ -354,3 +361,6 @@ To do so, it is advised to use Twister's hardware map feature with the following
 
 .. _STM32CubeProgrammer:
    https://www.st.com/en/development-tools/stm32cubeprog.html
+
+.. _STM32 ISP module:
+   https://github.com/stm32-hotspot/zephyr-stm32-mw-isp
```

Comment:
Even better would be to add the external module to https://docs.zephyrproject.org/latest/develop/manifest/index.html#external-projects-modules, wdyt?